<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-01-05T11:11:23 EST" name="Suite" finished-at="2025-01-05T11:11:41 EST" duration-ms="18221">
    <groups>
    </groups>
    <test started-at="2025-01-05T11:11:23 EST" name="Test" finished-at="2025-01-05T11:11:41 EST" duration-ms="18221">
      <class name="demo_qa.Verify_Dashboard">
        <test-method is-config="true" signature="initialization()[pri:0, instance:demo_qa.Verify_Dashboard@39655d3e]" started-at="2025-01-05T11:11:34 EST" name="initialization" finished-at="2025-01-05T11:11:37 EST" duration-ms="2998" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method signature="verify_SidePanel_List()[pri:1, instance:demo_qa.Verify_Dashboard@39655d3e]" started-at="2025-01-05T11:11:37 EST" name="verify_SidePanel_List" finished-at="2025-01-05T11:11:39 EST" duration-ms="2245" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_SidePanel_List -->
        <test-method signature="verify_Admin_Module_Heading()[pri:2, instance:demo_qa.Verify_Dashboard@39655d3e]" started-at="2025-01-05T11:11:39 EST" name="verify_Admin_Module_Heading" finished-at="2025-01-05T11:11:41 EST" duration-ms="1507" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Expected and Acrual is not match expected [Admin] but found [User Management]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Expected and Acrual is not match expected [Admin] but found [User Management]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at demo_qa.Verify_Dashboard.verify_Admin_Module_Heading(Verify_Dashboard.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Admin_Module_Heading -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:demo_qa.Verify_Dashboard@39655d3e]" started-at="2025-01-05T11:11:41 EST" name="tearDown" finished-at="2025-01-05T11:11:41 EST" duration-ms="188" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- demo_qa.Verify_Dashboard -->
      <class name="demo_qa.Login">
        <test-method is-config="true" signature="initialization()[pri:0, instance:demo_qa.Login@4f9a2c08]" started-at="2025-01-05T11:11:23 EST" name="initialization" finished-at="2025-01-05T11:11:28 EST" duration-ms="4747" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method signature="Verify_Login_with_Invalid_credential()[pri:1, instance:demo_qa.Login@4f9a2c08]" started-at="2025-01-05T11:11:28 EST" name="Verify_Login_with_Invalid_credential" finished-at="2025-01-05T11:11:30 EST" duration-ms="2015" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Login_with_Invalid_credential -->
        <test-method signature="Verify_user_can_log_in_with_valid_credential()[pri:2, instance:demo_qa.Login@4f9a2c08]" started-at="2025-01-05T11:11:30 EST" name="Verify_user_can_log_in_with_valid_credential" finished-at="2025-01-05T11:11:32 EST" duration-ms="1906" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_user_can_log_in_with_valid_credential -->
        <test-method signature="Verify_Logout()[pri:3, instance:demo_qa.Login@4f9a2c08]" started-at="2025-01-05T11:11:32 EST" name="Verify_Logout" finished-at="2025-01-05T11:11:34 EST" duration-ms="2120" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Logout -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:demo_qa.Login@4f9a2c08]" started-at="2025-01-05T11:11:34 EST" name="tearDown" finished-at="2025-01-05T11:11:34 EST" duration-ms="232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- demo_qa.Login -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
